Sources/Concrete/View/MultiHitsCollectionWidget.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  MultiHitsTablWidget.swift
+//  InstantSearch-iOS
+//
+//  Created by Guy Daher on 28/11/2017.
+//
+
+//
+//  MultiHitsTableWidget.swift
+//  InstantSearch-iOS
+//
+//  Created by Guy Daher on 23/11/2017.
+//
+
+import Foundation
+
+import Foundation
+import UIKit
+
+/// Widget that displays the search results of multi index. Built over a `UITableView`.
+@objc public class MultiHitsCollectionWidget: UICollectionView, MultiHitsViewDelegate, AlgoliaWidget {
+    
+    @IBInspectable public var showItemsOnEmptyQuery: Bool = Constants.Defaults.showItemsOnEmptyQuery
+    
+    @IBInspectable public var hitsPerSection: String = String(Constants.Defaults.hitsPerPage) {
+        didSet {
+            hitsPerSectionArray = hitsPerSection.components(separatedBy: ",").map({ (hitsPerSection) in
+                guard let hitsPerSectionUInt = UInt(hitsPerSection) else {
+                    fatalError("hitsPerSection should be comma seperated numbers")
+                }
+                
+                return hitsPerSectionUInt
+                }
+            )
+        }
+    }
+    
+    @IBInspectable public var indexNames: String = Constants.Defaults.indexName {
+        didSet {
```

Comment:
Why not applying the same logic as for indexIds: String ?

Reply:
indexNames have to be specified, else the Multi Hits Widget doesn't make sense anymore. The VM will throw the fatal error so that the developer can fix ths right away